README.md

```diff
@@ -54,25 +54,72 @@ The code is built using React-Native and running code locally requires a Mac or
 
 - [Check this link if you are having troubles with the command](https://www.rust-lang.org/tools/install)
 
-- Clone this repo and install our dependencies:
+- Clone this repo:
 
   ```bash
-  git clone ...
-  cd plug-mobile
-  yarn install
-  cd ios && pod install && cd .. # install pods for iOS
+  git clone repo-url
   ```
 
 - Follow the installation steps on our controller package called [plug-mobile-controller](https://github.com/Psychedelic/plug-mobile-controller).
 
+- Install our dependencies:
+
+```bash
+  cd plug-mobile
+  yarn install
+  cd ios && pod install && cd .. # install pods for iOS
+```
+
 - Choose the toolchain and install rust dependencies following the nexts steps:
 
   ```bash
   $ cargo install cargo-lipo
-  $ rustup default 1.55.0-aarch64-apple-darwin
+  $ rustup default 1.55.0-x86_64-apple-darwin
   $ rustup target add aarch64-apple-ios x86_64-apple-ios
+  $ rustup target add aarch64-linux-android
+  $ rustup target add armv7-linux-androideabi
+  $ rustup target add i686-linux-android
   ```
 
+- [Download NDK from Android Studio](https://developer.android.com/studio/projects/install-ndk), if you have M1 download the version `24.0.8215888`.
+- Install android-ndk:
+
+```bash
+brew install android-ndk
```

Comment:
i think we dont need this if we download it from Android Studio

Reply:
This is needed, we tried it first without this and didnt work

Reply:
But did you try changing also the NDK_HOME?
i deleted android-ndk from my computer